test/fixtures/account-token.js

```diff
@@ -0,0 +1,69 @@
+/*
+  This fixture creates a new application, directory,
+  and account.  It issues an Oauth request for the
+  account, resulting in an access token and refresh
+  token in the token collections for the account
+ */
+
+var helpers = require('../it/helpers');
+var OAuthPasswordGrantRequestAuthenticator = require('../../').OAuthPasswordGrantRequestAuthenticator;
+
+function AccountAccessTokenFixture(){
+  return this;
+}
+AccountAccessTokenFixture.prototype.before = function before(done) {
+  var self = this;
+  helpers.getClient(function(_client){
+
+    self.client = _client;
+    helpers.createApplication(function(err,app){
+      if (err) {
+        return done(err);
+      }else{
+        self.application = app;
+        helpers.getDefaultAccountStore(app,function(err,dir){
+          if (err) {
+            return done(err);
+          }else{
+            self.directory = dir;
+            self.newAccount = helpers.fakeAccount();
+            dir.createAccount(self.newAccount, function(err,_account) {
+              self.account = _account;
+              self.creationResult = [err, _account];
+
+              /*
+                We need to do a password grant request, so that we create
+                an access token and refresh token for this user
+               */
+
+              var authenticator = new OAuthPasswordGrantRequestAuthenticator(app);
+              authenticator.authenticate({
+                username: _account.username,
+                password: self.newAccount.password
+              },function(err,passwordGrantResult){
+                if(err){
+                  done(err);
+                }else{
+                  self.passwordGrantResult = passwordGrantResult;
+                  done();
+                }
+              });
+            });
+          }
+        });
+
+      }
+    });
+  });
+};
+AccountAccessTokenFixture.prototype.after = function after(done) {
+  var self = this;
```

Comment:
I do like this style, it works for the `after` method.  Do you know how to make it work with the `before` method, since we assign an object to the null value at a later time?